javascript/javascript3/week2/resolveAfterSettime.js

```diff
@@ -0,0 +1,10 @@
+function resolveAfterDelay(resolveAfter) {
+  const promise = new Promise((resolve, reject) => {
```

Comment:
We need to return the promise here. Otherwise we have no way of knowing when the 8 seconds have passed
Remember that asynchronous functions must always provide for us a way of detecting when the operation is done. Or else we can't really use it